core/src/main/java/org/apache/spark/ExecutorPlugin.java

```diff
@@ -40,12 +40,16 @@
    * Initialize the executor plugin.
    *
    * <p>Each executor will, during its initialization, invoke this method on each
-   * plugin provided in the spark.executor.plugins configuration.</p>
+   * plugin provided in the spark.executor.plugins configuration. The Spark executor
+   * will wait on the completion of the execution of the init method.</p>
    *
    * <p>Plugins should create threads in their implementation of this method for
    * any polling, blocking, or intensive computation.</p>
+   *
+   * @param pluginContext The user-supplied init method for this class is initialized
```

Comment:
The description of the parameter is redundant: we know it's a parameter to the init method; this is *the* documentation for that method. This should instead describe what that parameter is (e.g. "context information for the executor where the plugin is running").